erpnext/controllers/buying_controller.py

```diff
@@ -428,8 +429,9 @@ def validate_accepted_rejected_qty(self):
 			elif not flt(d.rejected_qty):
 				d.rejected_qty = flt(d.received_qty) -  flt(d.qty)
 
+			val  = flt(d.qty) + flt(d.rejected_qty)
 			# Check Received Qty = Accepted Qty + Rejected Qty
-			if ((flt(d.qty) + flt(d.rejected_qty)) != flt(d.received_qty)):
+			if (flt(val, float_precision) != flt(d.received_qty)):
```

Comment:
Rounding for `val` and `d.received_qty` should be based on same precision. I think in this we can do it based on precision of "received_qty" field instead of system settings precision.

Reply:
The received_qty has a field type float, which takes precision from system settings, on saving received qty will get rounded to the float precision from system settings. So shouldn't rounding be based on that float precision?